### PR TITLE
First round of CSS fixes

### PR DESCRIPTION
**Pull request by nok-ko**
Most things are better now, but I'll have to bash my head against font size.

Fixes:
- [X] Misplaced buttons
- [X] Goals being in a grid and kind of ugly
- [X] The wrapper not having extra padding to accommodate the sticky footer
- [X] Font size fixes for mobile – ~~I'm going to have a fun time with that one.~~ Fixed through the magic of friendship!

Important tweaks:
* EVERYTHING\* is flexbox now.
* Old `edit-2.svg`s are gone now, bring them back in another feature branch when we finally get image uploading
* 

Still to do:
- [ ] Betterize documentation and clean up the sheer mess in the CSS stylesheet.

### CODE REVIEW

File: feed.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <link rel="stylesheet" href="styles/common_styles.css">
+    <link rel="stylesheet" href="styles/feed_styles.css">
+</head>
+
+<body>
+    <main id="wrapper">
+        <section class="entry">
+            <!-- <img src="./images/" alt=""> -->
+            <!-- <h1 class="h3">User Bio</h1> -->
+            <!-- <p>Hi, I am Tommy Longname and this is my profile!</p> -->
```
Comment (mbabey) [inline feed.html:26]: Could prob dump this

**Comment by nok-ko** [comment]
Aaand font size fixed thanks to my friend's advice.

File: feed.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    </main>
+
+    <!--Footer-->
+    <footer class="sticky py-2 text-light">
+        <div role="group" class="meta-icon-container">
+            <div class="footer-icon-container" role="group">
+                <a href="profile.html">
+                    <img src="icons/user.svg#icon">
+                </a>
+                <!-- <span class="label">Profile</span> -->
```
Comment (mbabey) [inline feed.html:37]: Same for these comments

File: feed.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            </div>
+        </div>
+    </footer>
+
+    <!--Scripts-->
+    <script src="scripts/feed.js"></script>
+
+</body>
+
+</html>
```
Comment (mbabey) [inline feed.html:63]: newline

File: profile.html [hunk 4]
```diff
@@ -17,54 +17,55 @@
...
+                <img hidden id="saveInfo" src="icons/edit-2.svg#icon" onclick="saveProfile()" class="edit-icon"
+                    alt="Save">
+                <!-- <svg id="saveInfo" onclick="saveProfile()" class="edit-icon" aria-hidden="true" focusable="false">
                         <use xlink:href="icons/edit-2.svg#icon"></use>
                     </svg> -->
-                </div>
-                <img src="icons/edit-2.svg#icon" alt="test">
-          
+            </div>
+
```
Comment (mbabey) [inline profile.html:42]: No more edit button?

File: styles/profile_styles.css [hunk 5]
```diff
@@ -1,64 +1,76 @@
...
-    /* width: 1em;
-    height: 1em; */
+    width: 1em;
+    height: 1em;
     /* The default vertical-align is `baseline`, which leaves a few pixels of space below the icon. Using `center` prevents this. For icons shown alongside text, you may want to use a more precise value, e.g. `vertical-align: -4px` or `vertical-align: -0.15em`. */
     /* vertical-align: middle; */
     /* Paths and strokes that overflow the viewBox can show in IE11. */
     /* overflow: hidden; */
     cursor: pointer;
 }
```
Comment (mbabey) [inline styles/profile_styles.css:49]: We can scrap a lot of these commented out styles at this point. Not a thing on your branch, so its aight here

**Review by mbabey — APPROVED** [review]

File: profile.html (hunk 4 above)
Comment (nok-ko) [reply profile.html:42]: Add it back when you do image uploading, I don't want to deal with SVG headaches while the feature isn't even being worked on :P

File: styles/profile_styles.css (hunk 5 above)
Comment (nok-ko) [reply styles/profile_styles.css:49]: Yep, worth doing another general code cleanup after merging.

**Comment by nok-ko** [comment]
Oh, this also happens to fix Chi's calendar resizing in unexpected ways when hovered over.

**Comment by nok-ko** [comment]
Screenshot so far:
Screen Shot 2021-11-19 at 17 42 49

**Comment by nok-ko** [comment]
Future changes:
- [ ] Turn up global font size to make things less squinty on mobile
- [ ] Use variables for font size, and relative font sizes like `larger` & `smaller` etc.
- [ ] Actually implement the Feed feature (coming right up.)

**Pull request merged**